liquibase-standard/src/main/java/liquibase/Scope.java

```diff
@@ -113,21 +113,7 @@ public static Scope getCurrentScope() {
     }
 
     public static void setScopeManager(ScopeManager scopeManager) {
-        Scope currentScope = getCurrentScope();
```

Comment:
Besides the "default value" is removed here, the getCurrentScope later on has similar logic to make sure that we don't have a null scope.

Reply:
It is important to not inject common scope for scope manager, as all child will inherit from it, which you don't want if scope manager should have isolated scope.